jobs/kola-aws.Jenkinsfile

```diff
@@ -32,7 +32,11 @@ properties([
       string(name: 'COREOS_ASSEMBLER_IMAGE',
              description: 'Override the coreos-assembler image to use',
              defaultValue: "coreos-assembler:main",
-             trim: true)
+             trim: true),
+      string(name: 'CONFIG_GIT_COMMIT',
```

Comment:
Minor/optional: WDYT about renaming this `FCOS_CONFIG_COMMIT` everywhere instead? We can do that as a follow-up.

Reply:
> Minor/optional: WDYT about renaming this `FCOS_CONFIG_COMMIT` everywhere instead? We can do that as a follow-up.

i'm not opposed - want to open a follow-up PR?